<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveRangeDialog</class>
 <widget class="QDialog" name="RemoveRangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run Plugins</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="endFrameGroupBox">
     <property name="title">
      <string>end frame</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QSpinBox" name="endFrameSpinBox">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="endFrameSlider">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="endFrameImageLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="startFrameGroupBox">
     <property name="title">
      <string>start frame</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QSpinBox" name="startFrameSpinBox">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="startFrameSlider">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="startFrameImageLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Remove Annotations in Range</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>RemoveRangeDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startFrameSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>startFrameSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>endFrameSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>endFrameSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startFrameSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>startFrameSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>endFrameSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>endFrameSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
